src/components/Settings/ShareOnSocialMedia.js

```diff
@@ -74,65 +81,80 @@ class ShareOnSocialMedia extends Component {
   render() {
     return (
       <React.Fragment>
-        <DialogTitle>Share about SUSI.AI</DialogTitle>
-
-        <CopyWithButton
-          value={this.state.shareText}
-          width={this.props.width}
-          handleChange={this.handleChange}
-        />
-        <ShareIconContainer>
-          <IconButton>
-            <FacebookShareButton
-              url={this.state.shareUrl}
-              quote={this.state.title}
+        <SettingsTabWrapper heading="Share about SUSI.AI">
+          <CopyWithButton
+            value={this.state.shareText}
+            width={this.props.width}
+            handleChange={this.handleChange}
+          />
+          <ShareIconContainer>
+            <IconButton
```

Comment:
Make a common IconButton and reuse it, instead of repeating the code.

Reply:
I have made these changes. Please re-review.